src/components/Styles/Modal.styles.js

```diff
@@ -0,0 +1,69 @@
+import styled from 'styled-components'
+
+
+export const ModelStyle = styled.article`
+height: 578px;
+width: 640px;
+position: absolute;
+left: 590px;
+top: 200px;
+background: #FFFFFF;
+border-radius: 8px;
+text-align: center;
+background-color: " #FFFFFF";
+
+`
+export const HeadingStyle = styled.h1`
+font-family: 'Rubik';
+font-style: normal;
+font-weight: 600;
+font-size: 40px;
+margin: 48px 107px 39px 97px;
```

Comment:
Try to use reference properly. Either consistently use top margins or bottom margins.